examples/amp-story/attachment.html

```diff
@@ -57,6 +57,7 @@
           data-cta-text="Call To Action"
           cta-image="/extensions/amp-story/img/sweater.jpg"
           cta-image-2="/extensions/amp-story/img/boot.jpg"
+          theme="dark"
```

Comment:
nit:
For examples it's a good pattern to start with the most simple version, then show configurable options.
Let's put this on the second page.
Or, we could create a new example html document that's for showcasing our work on the new page attachment.
Each page could be a new feature/configuration. On the first page we could put instructions on how to turn the experiment on.

Reply:
yeah I think this makes sense. So I reverted the 'cover' page to be the existing page attachment UI, then the next 2 pages have the new UI + instructions on how to toggle it if you're not seeing it.